dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ExecutorServiceImpl.java

```diff
@@ -358,6 +358,7 @@ public Map<String, Object> execute(User loginUser, long projectCode, Integer pro
         ProcessDefinition processDefinition =
                 processService.findProcessDefinition(processInstance.getProcessDefinitionCode(),
                         processInstance.getProcessDefinitionVersion());
+        processDefinition.setReleaseState(ReleaseState.ONLINE);
```

Comment:
It's better remove the release state checking logic since this check is meaningless if `setReleaseState(ReleaseState.ONLINE)` are added.
https://github.com/apache/dolphinscheduler/blob/0464123c2b8d507a16a966e2dbe8dbfe9bf781e2/dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ExecutorServiceImpl.java#L277-L279

Reply:
For heavy running and other movements, it is no problem to ignore them like this